components/login/login.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { Button } from "@chakra-ui/react";
+import { GoogleAuthProvider, signInWithPopup } from "firebase/auth";
+import { auth } from "../../config/firebaseConfig";
+import { useAuth } from "../../context/authContext";
+
+export const Login = () => {
+  const { user, setUser }: any = useAuth();
+
+  const googleProvider = new GoogleAuthProvider();
+
+  function handleClick() {
+    if (user.uid) {
+      console.log("Already Logged In");
+    } else {
+      signInWithPopup(auth, googleProvider)
+        .then(({ user }) => {
+          console.log(user);
+          const userData = {
+            name: user.displayName,
+            email: user.email,
+            photo: user.photoURL,
+            uid: user.uid,
+          };
+          localStorage.setItem("auth", JSON.stringify(userData));
+          setUser(userData);
+        })
+        .catch((error) => {
+          console.log(error);
+          return;
+        });
+    }
+  }
+
+  return (
+    <div>
```

Comment:
When you use Chakra UI, make use of `Box` or other pre build components,
avoid using HTML elements.

Reply:
Yeah, not started working on the UI, I'll keep it in mind once I start working.
Thanks for the suggestion